### ISSUE TITLE
Training Details

### ISSUE THREAD

**Issue by wenbinTse** [pkuxmq/Invertible-Image-Rescaling#7]
Hi, thank for your amazing work.
I want to reproduce your work using the config `train_IRN_x2.yml`, but the training is slow and will cost 4 days using the default config. Do I need to notice any detail? BTW, does the default config in yml file is the correct one to reproduce the paper results?

**Comment by pkuxmq** [comment]
Yes it will cost these time to train 500k iterations. The default config is the correct one.

**Comment by ManWingloeng** [comment]
Can you reproduce the DIV2k_val psnr? I train for 3 days and I think I cannot get the 44 psnr.
![image](https://user-images.githubusercontent.com/20945688/97033676-4b726d00-1596-11eb-859c-8eea288a0d3e.png)
@wenbinTse @pkuxmq

**Comment by ManWingloeng** [comment]
> Can you reproduce the DIV2k_val psnr? I train for 3 days and I think I cannot get the 44 psnr.
> ![image](https://user-images.githubusercontent.com/20945688/97033676-4b726d00-1596-11eb-859c-8eea288a0d3e.png)
> @wenbinTse @pkuxmq

Is that the tensorboard show the PSNR and the paper show the PSNR_Y? @pkuxmq

**Comment by pkuxmq** [comment]
@ManWingloeng Yes, the training code only calculates RGB PSNR, and the testing code will calculate both RGB PSNR and Y PSNR. We report Y PSNR in the paper following the setting of other works. For IRN_x2, the final RGB PSNR should be around 40.8dB.

**Comment by ManWingloeng** [comment]
> @ManWingloeng Yes, the training code only calculates RGB PSNR, and the testing code will calculate both RGB PSNR and Y PSNR. We report Y PSNR in the paper following the setting of other works. For IRN_x2, the final RGB PSNR should be around 40.8dB.

Thanks your reply.
